.flutter-plugins-dependencies

```diff
@@ -0,0 +1 @@
"plugins":{"ios":[{"name":"connectivity_plus","path":"/Users/ktruon/.pub-cache/hosted/pub.dev/connectivity_plus-4.0.2/","native_build":true,"dependencies":[]},{"name":"device_info_plus","path":"/Users/ktruon/.pub-cache/hosted/pub.dev/device_info_plus-9.0.3/","native_build":true,"dependencies":[]},{"name":"package_info_plus","path":"/Users/ktruon/.pub-cache/hosted/pub.dev/package_info_plus-4.1.0/","native_build":true,"dependencies":[]}],"android":[{"name":"connectivity_plus","path":"/Users/ktruon/.pub-cache/hosted/pub.dev/connectivity_plus-4.0.2/","native_build":true,"dependencies":[]},{"name":"device_info_plus","path":"/Users/ktruon/.pub-cache/hosted/pub.dev/device_info_plus-9.0.3/","native_build":true,"dependencies":[]},{"name":"flutter_plugin_android_lifecycle","path":"/Users/ktruon/.pub-cache/hosted/pub.dev/flutter_plugin_android_lifecycle-2.0.15/","native_build":true,"dependencies":[]},{"name":"package_info_plus","path":"/Users/ktruon/.pub-cache/hosted/pub.dev/package_info_plus-4.1.0/","native_build":true,"dependencies":[]}],"macos":[{"name":"connectivity_plus","path":"/Users/ktruon/.pub-cache/hosted/pub.dev/connectivity_plus-4.0.2/","native_build":true,"dependencies":[]},{"name":"device_info_plus","path":"/Users/ktruon/.pub-cache/hosted/pub.dev/device_info_plus-9.0.3/","native_build":true,"dependencies":[]},{"name":"package_info_plus","path":"/Users/ktruon/.pub-cache/hosted/pub.dev/package_info_plus-4.1.0/","native_build":true,"dependencies":[]}],"linux":[{"name":"connectivity_plus","path":"/Users/ktruon/.pub-cache/hosted/pub.dev/connectivity_plus-4.0.2/","native_build":false,"dependencies":[]},{"name":"device_info_plus","path":"/Users/ktruon/.pub-cache/hosted/pub.dev/device_info_plus-9.0.3/","native_build":false,"dependencies":[]},{"name":"package_info_plus","path":"/Users/ktruon/.pub-cache/hosted/pub.dev/package_info_plus-4.1.0/","native_build":false,"dependencies":[]}],"windows":[{"name":"connectivity_plus","path":"/Users/ktruon/.pub-cache/hosted/pub.dev/connectivity_plus-4.0.2/","native_build":true,"dependencies":[]},{"name":"device_info_plus","path":"/Users/ktruon/.pub-cache/hosted/pub.dev/device_info_plus-9.0.3/","native_build":false,"dependencies":[]},{"name":"package_info_plus","path":"/Users/ktruon/.pub-cache/hosted/pub.dev/package_info_plus-4.1.0/","native_build":false,"dependencies":[]}],"web":[{"name":"connectivity_plus","path":"/Users/ktruon/.pub-cache/hosted/pub.dev/connectivity_plus-4.0.2/","dependencies":[]},{"name":"device_info_plus","path":"/Users/ktruon/.pub-cache/hosted/pub.dev/device_info_plus-9.0.3/","dependencies":[]},{"name":"package_info_plus","path":"/Users/ktruon/.pub-cache/hosted/pub.dev/package_info_plus-4.1.0/","dependencies":[]}]},"dependencyGraph":[{"name":"connectivity_plus","dependencies":[]},{"name":"device_info_plus","dependencies":[]},{"name":"flutter_plugin_android_lifecycle","dependencies":[]},{"name":"package_info_plus","dependencies":[]}],"date_created":"2023-08-17 17:11:41.266673","version":"3.10.6"}
```

Comment:
This file should not be tracked. Needs to be removed from the cache (`git rm --cached .flutter-plugin-dependencies`) and added to the gitignore.

Reply:
The problem I think is that you've run `flutter pub get` in this project, even though it is Dart only, and the Dart `gitignore` and Flutter `gitignore` vary.